sdks/java/container/build.gradle

```diff
@@ -77,7 +77,9 @@ golang {
 import com.github.jk1.license.render.*
 licenseReport {
   outputDir = 'java_third_party_licenses'
-  projects = [project.rootProject] + project.rootProject.subprojects
+  projects = [project,
+              project.rootProject.findProject(':sdks:java:harness'),
+              project.rootProject.findProject(':sdks:java:io:kafka')]
```

Comment:
Does this change include all dependencies installed in container? If so, it looks good for now.
This list should evolve if more dependencies are added to container, so we need to make it clear somewhere for developers who add new dependencies to container. 

A minor update to [release guide](https://github.com/apache/beam/blob/master/website/www/site/content/en/contribute/release-guide.md) is needed. There is a step to check if all licenses are added by checking the number of licenses. Now we expect to see roughly 1400 dependencies. With this change, I think the number is reduced? Then the guide should be updated with a new number or find a better way to check if all expected licenses are included.

Reply:
Hmmm. When I ran `:sdks:java:container:generateLicenseReport` before this PR I got 50 dependencies. After this PR I got 55. See the diff in the description. It mostly added new deps. The ones removed were:

 - `checker-qual-3.4.1.jar`: this is `compileOnly` so we don't need it... but it should be in the configuration I chose...
 - `hadoop-apache2-3.2.0-1.jar`: I hope this is not in the container!
 - `icu4j-58.2.jar`: I don't know
 - `objenesis-2.6.jar`: I don't know
 - `woodstox-core-asl-4.4.1.jar`: I don't know

But it sounds like there's another problem because I should be able to repro and get 1400 licenses at head.

Reply:
I see. I need to run the wrapper script to further process the output of this command. I expect it will be monotonic (more output from generateLicenseReport --> more licenses in the end) but I will double check and do a full diff of the larger output.

Reply:
Yep, need to run `pullLicenses` task with  `--docker-pull-licenses` options. `generateLicenseReport` task pulls licenses from jars. If licenses are not distributed within jar, they are ignored. `pullLicenses` task pulls those missing licenses, so will have more licenses. But I don't think the gap should be 50 vs 1400.